metricbeat/module/etcd/test_etcd.py

```diff
@@ -11,7 +11,7 @@ class Test(metricbeat.BaseTest):
     COMPOSE_SERVICES = ['etcd']
 
     @parameterized.expand([
-        "leader",
+        # "leader",
```

Comment:
Why is this commented?

Reply:
`leader` metricset events are triggered if the etcd instance has more than 0 followers - the testsuite starts etcd with 0 followers therefore no events will be found and tests will fail.